haiku-apps/sync_modular/sync_modular-2.2.1b.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="Native, modular software synthesizer"
+DESCRIPTION="It gives you, though limited as any synth, but quite versatile possibilities for creating \
+and using new sounds.
+
+Registration:
+User name : Free registration
+Serial number : eCDiPa-kbbmPZ-zGDJuyU
+
+Here we are going to list the most important features of SynC Modular.
+
+* SynC Modular is modular. This means that it has no fixed structure. Instead it allows \
+you to create your own synth structures, as well as using the synth structures created \
+by others. Ideally you can simulate any synthesizer with its modular features. In \
+practice, though, there are always limits to all vanilla promises.
+
+* SynC Modular is multitimbral. You can have as many as 16 different synths playing \
+at the same time within SynC Modular. Besides that you can change any of those \
+synths for another one on the fly, by means of MIDI program change. The limit to this \
+vanilla promise is your CPU performance which sets the top margin on the overall \
+complexity of the synths.
+
+* SynC Modular is fast. With its unique modular optimization technology it offers you \
+the performance of a non-modular software synth.
+
+* SynC Modular is flexible. When going down the module structure it allows you as deep \
+as creating your own oscillators and filters! The Sawtooth oscillator, the Sine oscillator, \
+the 1- and 2-pole filters and many more modules are not the basic pieces. You can dig \
+further into their internal structure. This promise is limited by the fact that you better \
+know the DSP theory if you are willing to design your own filters, rather than combining \
+the existing ones. But hey, you could use filters created by someone else, coudn't you?"
+HOMEPAGE="http://www.mtu-net.ru/syncmodular"
+COPYRIGHT="1999-2000, Dr Sync"
+LICENSE="Sync_Modular"
+REVISION="1"
+SOURCE_URI="http://www.sync-modular.org/download2_BeOS.php"
+CHECKSUM_SHA256="a29bf21d50822106cf4d2b103ca42634f132c1e12d97959d82deb080c564e19e"
+
+ARCHITECTURES="x86_gcc2 !x86"
+SECONDARY_ARCHITECTURES="x86_gcc2"
+
+DISABLE_SOURCE_PACKAGE=yes
+
+PROVIDES="
+	sm$secondaryArchSuffix = $portVersion
+	app:sync_modular = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	"
+BUILD_PREREQUIRES="
```

Comment:
**Excellent advice!** I knew that it is recommended to use **`cmd:`** entries but was too lazy to mention it. So thanks @pulkomandy for your explanation!
@miqlas, could you try to build the recipe without the **`coreutils`** line? If it complains about one or more missing commands and you find out that these are in the coreutils package, then you can safely add **`cmd:`** entries for them.
Reminder: if a recipe needs two commands, say **`cmd:foo`** and **`cmd:bar`**, and if you add only one of them to the recipe, say **`cmd:foo`**, and if that **`cmd:foo`** is only available in a package that also provides the other command, **`cmd:bar`**, then you won't notice that the recipe also needs **`cmd:bar`** because that command will be pulled together with all **`cmd:`** entries provided by the package that provides **`cmd:foo`**. In other words, you can't assume you got all required **`cmd:`** entries in a recipe's BUIILD_PREREQUIRES just because it builds fine. You should, ideally, try to identify all needed **`cmd:`** entries by examining carefully the output of the build. I usually have a look at the **`configure.ac`** file. Then, when I kick the build, I keep an eye on the output of the configure script until it finishes its job.